src/pages/help-centre/dbot.js

```diff
@@ -73,7 +54,8 @@ const ImageWrapper = styled.div`
 `
 
 const ImageWrapperRemove = styled.div`
-    margin-left: auto;
+    margin-top: 2.4rem;
+    margin-left: 7rem;
```

Comment:
```suggestion
    margin: 2.4rem 0 0 7rem;
```

Reply:
I guess it's better to specify the margin since there is no relation between `top - left`. this is to avoid assigning `bottom - top` to their default values.

If there was one more for either `bottom - top` I would go with `margin: ...`. 😂